phase1.js

```diff
@@ -1,24 +1,50 @@
-
 const puppeteer = require('puppeteer');
+const fs = require('fs');
 
 (async () => {
   const browser = await puppeteer.launch();
   const page = await browser.newPage();
 
-  await page.goto('https://advantedgeadvisors.com/investment-banker-articles/page/4/', {
-    waitUntil: 'networkidle2',
-  });
-  let data = await page.evaluate(()=> {
-      const list = document.querySelectorAll('div[class="elementor-post__card"] > a');
-      const anchors = Array.from(list);
-      const hrefs = anchors.map( a => a.href);
-
-      return { hrefs }
-  })
-  console.log(data)
+  let nextPage = 'https://advantedgeadvisors.com/investment-banker-articles';

```

Comment:
Instead of modifying the code and scraping one page at a time, we can start on the main index page here...